lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -106,23 +106,24 @@ Future<Uint8List> fetchImage(
     String url, WebOnlyImageCodecChunkCallback? chunkCallback) {
   final Completer<Uint8List> completer = Completer<Uint8List>();
 
-  final html.HttpRequest request = httpRequestFactory();
-  request.open('GET', url, async: true);
+  final DomXMLHttpRequest request = httpRequestFactory();
+  request.open('GET', url, /*async*/ true);
```

Comment:
If we're just changing this to a positional argument let's just omit the `/*async*/`